usr/share/rear/prep/OPALPBA/Linux-i386/001_configure_workflow.sh

```diff
@@ -29,6 +29,7 @@ if (( ${#OPAL_PBA_PROGS[@]} == 0 && ${#OPAL_PBA_COPY_AS_IS[@]} == 0)) && has_bin
     LogPrintError "     interface for the PBA by setting OPAL_PBA_{PROGS,COPY_AS_IS,LIBS} to include Plymouth components."
 fi
 PROGS+=( "${OPAL_PBA_PROGS[@]}" clear )
+[[ -n "$OPAL_PBA_DEBUG_PASSWORD" ]] && PROGS+=( openssl )
```

Comment:
Shouldn't that better be `.... REQUIRED_PROGS+=( openssl )`
to ensure an `openssl` program must be there in the recovery system
or do things also work when `openssl` is missing in the recovery system?

Reply:
Yes, it would work (i.e. silently fail), and `openssl` is included in ReaR's package requirements (at least on Debian systems) but I agree that using `REQUIRED_PROGS` is a better solution.